datastore/gcloud/rest/datastore/value.py

```diff
@@ -61,7 +64,7 @@ def from_repr(cls, data):
     def to_repr(self):
         # type: () -> Dict[str, Any]
         value_type = self._infer_type(self.value)
-        if value_type == TypeName.KEY:
+        if value_type in [TypeName.GEOPOINT, TypeName.KEY]:
```

Comment:
Use a set here rather than a list, that'll be (technically) a bit faster and feels a bit more semantically correct.